docs/source/concept_models/vivarium_swissre_colorectalcancer/concept_model.rst

```diff
@@ -479,49 +506,43 @@ Relevant formulas
      - Source
      - Note
    * - MST
-     - 4.5-5.8 years
+     - 4.5-5.8 years depending on age
      - [Brenner-et-al-2012]_
-     - 
+     - Let's use a MST of 5 years for all ages.
    * - Colon and rectum cancer screening coverage in baseline
-     - 20%
+     - 15%
      - [lin-et-al-2019]_
      - 
    * - Colon and rectum cancer screening covearge in alternative scenario
      - scale up to 60% in 2030
      - [de-Moor-et-al-2018]_
-     - We applied 20-year lag on US coverage
+     - We applied a 20-year lag on US coverage.
    * - FOBT sensitivity
      - 68% (95%CI 57%-79%)
      - [Meklin-et-al-2020]_
-     - We documented the sensitivity of gFOBT
+     - Normal distribution with mean=0.68 and std_dev=0.056
    * - FOBT specificity
      - 88% (95%CI 84%-91%)
      - [Meklin-et-al-2020]_
-     - We documented the specificity of gFOBT
+     - Normal distribution with mean=0.88 and std_dev=0.018
    * - Colonoscopy sensitivity
      - 98% (95%CI 91%-100%)
      - [Rockey-et-al-2005]_
-     - 
+     - Triangular distribution with left=0.90, mode=0.98, and right=1.00
    * - Colonoscopy specificity
      - 100%
      - 
      - by client’s assumption (no false positive results of screening)
-   * - Prevalence of family history of colon and rectum cancer
-     - 3-10%
-     - [Henrikson0-et-al-2015]_
-     - 
-   * - Relative risk of family history of colon and rectum cancer
-     - 1.76 (95%CI 1.57-1.97)
-     - [Wong-et-al-2018]_
-     - 
-   * - Prevalence of personal history of adenoma
-     - 
-     - 
-     - 
-   * - Relative risk of personal history of adenoma
-     - 2.7 (95%CI, 1.9-3.7)
-     - [Click-et-al-2018]_
-     - 
+   * - Prevalence of having family history of CRC or personal history of 
+       adenoma, or both
+     - Normal distribution with mean=0.05 and std_dev=0.005
```

Comment:
@collijk : is there a way to specify a normal distribution for the exposure in the `model_spec.in` file?  I can't figure it out, and I'm going to use 
```
    family_history_or_adenoma:
        exposure: 0.05
```
for now.